firefox-ios/Client/Frontend/Browser/SearchViewController.swift

```diff
@@ -241,7 +242,7 @@ class SearchViewController: SiteTableViewController,
 
     /// Whether to show sponsored suggestions from Firefox Suggest.
     private var shouldShowSponsoredSuggestions: Bool {
-        shouldShowNonSponsoredSuggestions &&
+        return !viewModel.isPrivate &&
```

Comment:
just confirm, we only want to show sponsor items in non private mode?

Reply:
Exactly!!